web/src/stores.ts

```diff
@@ -65,6 +65,8 @@ export let currentProjectKey: Writable<string> = writable("");
 // False until user activates
 export let showAbout: Writable<boolean> = writable(false);
 
+export let appFocus: Writable<"global" | "cnt"> = writable("global");
```

Comment:
Context is hard to access outside of a component, so I moved it to a store. If you prefer, we could instead try a `ChooseProjectLink` component, so that context is visible

Reply:
A store seems fine!